Lib/asyncio/base_events.py

```diff
@@ -926,6 +992,59 @@ def _check_sendfile_params(self, sock, file, offset, count):
 
         return transport, protocol
 
+    async def sendfile(self, transport, file, offset=0, count=None,
+                       *, fallback=True):
+        """Send a file through a transport.
+
+        Return amount of sent bytes.
+        """
```

Comment:
Copy the relevant snippet from the documentation to make it a proper docstring.

Reply:
Did it but I feel that docstring is too long.